release/release_build.py

```diff
@@ -66,6 +67,7 @@ def clone_or_update_repos(auto):
 
     clone_from_scratch_or_update(LIVE_PLUME_LIB, PLUME_LIB, clone_from_scratch, False)
     clone_from_scratch_or_update(LIVE_PLUME_BIB, PLUME_BIB, clone_from_scratch, False)
+    clone_from_scratch_or_update(LIVE_STUBPARSER, STUBPARSER, clone_from_scratch, False)
```

Comment:
This clones the stub parser, but doesn't build it.  (Plume lib is built when it is cloned in the post-merge hook.)

Reply:
I'm working on building it.

Reply:
Added in 28c101e.